benchmarks/raw_test.bench

```diff
@@ -0,0 +1 @@
+{"Benchmark":["ExampleSHA256",3]}
```

Comment:
Is it possible to add a comment here to specify what the key and every value in the list is?

Reply:
Yes, that would be ok.